modules/nw-about-automatic-egress-ips.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/enabling-automatic-egress-ips.adoc
+
+[id="about-automatic-egress-ips-{context}"]
+= About automatic egress IPs
+
+You can assign egress IP addresses to namespaces by setting the `egressIPs`
+field to the `NetNamespace` resource. With fully automatic egress IPs, you can
```

Comment:
"the `egress IPs` field **of** the `NetNamespace` resource". Or maybe "on".

Reply:
I don't know how the different pieces of the 4.1 docs fit together, but it seems like you're mentioning "fully-automatic egress IPs" here without having explained what that means. (It's possible the 3.x docs were bad about this as well.)

(There are "manually-assigned" (or "semi-automatic") egress IPs, where the administrator explicitly picks which egress IPs will be on which nodes, and so has to set `egressIPs` on both the `NetNamespace` and the `HostSubnet` objects. And then there are "automatically-assigned" or "fully-automatic" egress IPs, where the administrator sets `egressIPs` on `NetNamespace` but sets `egressCIDRs` on the `HostSubnet`, and then OpenShift figures out the assignment of specific egress IPs to nodes.)

The manual/semi-automatic mode is older and less useful, so it makes sense for the docs to primarily focus on the fully-automatic mode, but we need to document the semi-automatic mode too.

Reply:
I have a separate PR for the manually assigned approach (https://github.com/openshift/openshift-docs/pull/14555). It may make more sense for the content for both PRs to be in a single assembly, with the fully automatic content followed by the manually assigned content.

All of this was derived from the monolithic network configuration page for OCP 3.x.

Reply:
@danwinship, what is the `NetNamespace` and what is the `HostSubnet`? It seems like a namespace itself may be synonymous with a _project_, so is the `NetNamespace` related to the network configuration for a project? And the `HostSubnet` relates to nodes available in the cluster?

Thanks.

Reply:
Yeah, `NetNamespace` is the object that OpenShift SDN uses to store its own per-namespace data, and `HostSubnet` is the object it uses to store its own per-node data. So the `NetNamespace` will have the same name as the namespace/project you're using an egress IP for, and the `HostSubnet` will have the same name as the node you're assigning the egress IP to.